### PR TITLE
feat(frontend): allow user to specify the min number of VCPU and RAM

### PR DESCRIPTION
**Pull request by zhouzilong2020**
This PR should follow #7 
截屏2022-04-25 10 13 35

### CODE REVIEW

**Comment by zhouzilong2020** [comment]
PTAL

File: frontend/src/components/CostTable.vue [hunk 1]
```diff
@@ -182,11 +188,32 @@ watch(
...
+  }
+);
+
 const refreshDataTable = () => {
   state.dataRow = [];
   let rowCnt = 0;
   const selectedRegionSet = new Set<string>([...props.regionList]);
   props.dbInstanceList.forEach((dbInstance) => {
+    if (
+      Number(dbInstance.memory) < props.minRAM ||
```
Comment (d-bytebase) [inline frontend/src/components/CostTable.vue:211]: qq: any reason dbInstance.memory wasn't Number?

**Review by d-bytebase — APPROVED** [review]

File: frontend/src/components/CostTableMenu.vue [hunk 2]
```diff
@@ -14,43 +14,73 @@
     </n-checkbox-group>
   </div>
 
-  <div class="mt-4 space-x-2 flex justify-start">
+  <div class="mt-2 flex flex-wrap">
+    <!-- Min specification for Memory & vCPU -->
+    <div class="inline-block align-bottom max-w-min mr-2 pt-2">
+      <n-input-number
```
Comment (tianzhou) [inline frontend/src/components/CostTableMenu.vue:20]: this control doesn't feel right to input the number directly, there is no input cursor. My feeling is it's more used for changing the amount (like a ticker used in the shopping cart).

I would suggest to use a normal text input only accepting integers or just use a dropdown with some predefined values (e.g 1, 2, 4, 8 and etc)

File: frontend/src/components/CostTable.vue (hunk 1 above)
Comment (zhouzilong2020) [reply frontend/src/components/CostTable.vue:211]: Thia may be the consequence of contributing without code review😢. 
I will fix this by another PR.

File: frontend/src/components/CostTableMenu.vue (hunk 2 above)
Comment (zhouzilong2020) [reply frontend/src/components/CostTableMenu.vue:20]: I slightly change the style, making the +/- button disappear and showing user the input cursor.
wdyt?

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**